docs/sources/metainfo/component.xml

```diff
@@ -143,6 +143,32 @@
 		</listitem>
 		</varlistentry>
 
+		<varlistentry id="tag-icon">
+		<term>&lt;icon/&gt;</term>
+		<listitem>
+			<para>
+				The <code>&lt;icon/&gt;</code> tag describes the component icon. It is mostly used for GUI applications (component-type <literal>desktop-application</literal>).
+				It can be of the type <literal>stock</literal>, <literal>local</literal>,
+				or <literal>url</literal>.
+			</para>
+			<para>
+				<literal>stock</literal> icons are loaded from stock. The icon name should never include any file-extension or path.
+			</para>
+			<para>
+				<literal>local</literal> icons are loaded from a file in the filesystem.
+				They should specify a full file path in either <code>/usr/share/pixmaps</code>
+                                or <code>/usr/share/icons</code>. (These restrictions on the icon location are in
+                                place to allow more efficient creation of collection meta data.)
+				This icon type may have <literal>width</literal> and <literal>height</literal>
+                                properties.
+			</para>
+			<para>
+				<literal>remote</literal> icons loaded from a remote URL. Currently, at least HTTP urls must be supported.
+				This icon type may have <literal>width</literal> and <literal>height</literal> properties.
```

Comment:
Make that "may" a "should".

Reply:
Changed. I have also made the corresponding change in xmldata.xml